src/mbgl/style/style_impl.hpp

```diff
@@ -86,10 +86,12 @@ class Style::Impl : public SpriteLoaderObserver,
     Immutable<std::vector<Immutable<Layer::Impl>>> getLayerImpls() const;
 
     void dumpDebugLogs() const;
+    bool areSpritesLoaded() const;
 
     bool mutated = false;
     bool loaded = false;
-    bool spriteLoaded = false;
+    bool countOfSprites = 0;
```

Comment:
Probably a `size_t` as intended here.

Reply:
Loading of sprites are now managed with a map of ids/booleans to dodge possible issues with multiple calls to onSpriteLoaded.